src/telcameras_v2/models.py

```diff
@@ -1,8 +1,12 @@
 from django.contrib.postgres import fields as postgres_fields
 from django.db import models
+from django.utils import timezone
+
+from contrib.timescale.fields import TimescaleDateTimeField
 
 
 class Observation(models.Model):
+    time = TimescaleDateTimeField(interval="1 day", default=timezone.now)
```

Comment:
Hier kan je de bestaande `timestamp_start` voor gebruiken (dat veld kan je eventueel hernoemen naar `timestamp`). Dan is het veld `observation_timestamp` ook weer logischer in de twee aggregate models (want observation.timestamp)